clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceAfterImageConsumerImpl.java

```diff
@@ -134,9 +133,18 @@ public CompletableFuture<Void> seekToEndOfPush(Set<Integer> partitions) {
           }
 
           // poll until we get EOP for all partitions
+          LOGGER.info("Polling for EOP messages for partitions: " + partitions.toString());
           synchronized (consumerAdapter) {
+            LOGGER.info("GOT LOCK");
+            int counter = 0;
             while (true) {
+              counter++;
+              if (counter > 500) {
+                LOGGER.info("This is taking a while.....");
```

Comment:
Should we add a bit more info for this log? e.g. the partitions we are still waiting for in `endOfPushConsumedPerPartitionMap`?

Reply:
oh crap, let me remove those, they were something I was putting in for debugging